test/docs/subprocess_test.rb

```diff
@@ -89,4 +89,40 @@ class Memo::Create < Trailblazer::Activity::Railway
     end
   end
 
+  it "subprocess automatically wires all termini of a nested activity" do
+    module D
+      Memo = Class.new
+
+      #:end-nested
```

Comment:
`#:end-nested` must be unique in given file. Otherwise it overrides existing code while updating website doc.

Reply:
@yogeshjain999  oh, you are right. I'll fix this